src/components/Icon/index.examples.js

```diff
@@ -64,6 +64,13 @@ storiesOf('Icons')
     <Icon type='lesson' />
   ))
 
+  .addWithInfo('Refresh', () => (
+    <Icon
+      type='refresh' 
+      spin
+    />
+  ))
```

Comment:
@tayiorbeii That's cool, but you could still do it programmatically rather than manually :) This works fine for now though.